cmd/goal/accountsList.go

```diff
@@ -231,6 +231,21 @@ func (accountList *AccountsList) outputAccount(addr string, acctInfo v1.Account,
 		}
 		fmt.Printf("]")
 	}
```

Comment:
I know it's not part of the PR but maybe worth fixing it now. Just add the word ID -> `Created assets IDs`.

Reply:
done